### PR TITLE
Update HTML Symbols

### PR DESCRIPTION
**Pull request by james-k-7**
**Description**

### CODE REVIEW

**Comment by Bouncey** [comment]
@Nirajn2311 Care to elaborate? I don't know what you mean?

File: src/pages/html/symbols/index.md [hunk 1]
```diff
@@ -2,14 +2,56 @@
...
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/symbols/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+To add such symbols to an HTML page, you can use an HTML entity name.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+If no entity name exists, you can use an entity number, a decimal, or hexadecimal reference.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+### Examples
+#### A Few Mathematical Symbols
```
Comment (Nirajn2311) [inline src/pages/html/symbols/index.md]: The tittle should be “A Few Greek Letters”.

File: src/pages/html/symbols/index.md [hunk 2]
```diff
@@ -2,14 +2,56 @@
...
+↑	&#8593;	&uarr;	  UPWARDS ARROW
+→	&#8594;	&rarr;	  RIGHTWARDS ARROW
+↓	&#8595;	&darr;	  DOWNWARDS ARROW
+♠	&#9824;	&spades;  BLACK SPADE SUIT
+♣	&#9827;	&clubs;	  BLACK CLUB SUIT
+♥	&#9829;	&hearts;  BLACK HEART SUIT
+♦	&#9830;	&diams;   BLACK DIAMOND SUIT
+```
 
+#### A Few Greek Letters
```
Comment (Nirajn2311) [inline src/pages/html/symbols/index.md]: The title should be “A Few Mathematical Symbols”

**Comment by QuincyLarson** [comment]
Thank you for your extreme patience. We were overwhelmed with pull requests last October, and have spent the past year getting caught up.

We now have a full team of moderators who are going to review future pull requests.

We are archiving the Guide repo itself in favor of moving the Guide over to our main github.com/freecodecamp/freecodecamp repo. 

If this pull request still improves upon the Guide, I encourage you to re-open your pull request over on the new repo.

Thanks again for your understanding. I hope you'll consider contributing to the newly updated guide.

**Pull request closed without merging**